src/main/java/com/boardgo/domain/termsconditions/entity/UserTermsConditionsEntity.java

```diff
@@ -0,0 +1,41 @@
+package com.boardgo.domain.termsconditions.entity;
+
+import com.boardgo.domain.termsconditions.entity.enums.TermsConditionsType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Comment;
+
+@Entity
+@Getter
+@Table(name = "user_terms_conditions")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class UserTermsConditionsEntity {
+
+    @Id
+    @Column(name = "user_terms_conditions_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Comment("회원 고유 id")
+    @Column(name = "user_info_id", nullable = false)
+    private Long userInfoId;
+
+    @Comment("약관동의 타입")
+    @Enumerated(EnumType.STRING)
+    @Column(name = "terms_conditions_type", length = 20, nullable = false)
+    private TermsConditionsType termsConditionsType;
```

Comment:
약관동의 타입이 두 개 있는데 이유가 있는지 궁금합니다!

Reply:
아 약관동의 엔티티에 `type`랑 같은 타입인데, 이름만 다르게했어요
칼럼명 같게 수정하는게 나을까요?
```suggestion
    @Comment("약관동의 타입")
    @Enumerated(EnumType.STRING)
    @Column(name = "type", length = 20, nullable = false)
    private TermsConditionsType termsConditionsType;
```

생각해보니 회원 약관동의-약관동의 엔티티가 `@ManyToOne` 이 성립하는 것 같지 않나요?
(여러명의 회원이 하나의 약관동의를 선택하니까)
맞다면 `@ManyToOne` 를 추가하고 지금 코드대로 하는건 어떤가요?

Reply:
어 .. 근데 엔티티에도 타입이 있고 지금 유저 약관엔티티에도 두 개가 있는게 의도가 맞으신가요?

Reply:
네 맞아요! 회원의 약관동의를 저장할때 약관동의의 타입을 저장하려고 했어요

Reply:
아 그러면 타입만 조회하는 경우가 많아서 저렇게 구성하신건가요 ?

Reply:
만약에 약관 동의를 조회하는 곳이 있다면 조회할 때도 필요하고, 회원이 어떤 약관을 동의했는지 저장해야 하니까 약관동의 고유키 말고 타입으로 구성했어요